rpcs/ledgerService.go

```diff
@@ -189,6 +195,11 @@ func (ls *LedgerService) ServeHTTP(response http.ResponseWriter, request *http.R
 		}
 	}
 	defer cs.Close()
+	response.Header().Set("Content-Type", LedgerResponseContentType)
+	if request.Method == http.MethodHead {
+		response.WriteHeader(http.StatusOK)
+		return
+	}
```

Comment:
This ServeHTTP function is wild... I'm starting to see what you were talking about.

Reply:
Also, nice. This is a great solution, I didn't realize you were implementing across the client and the server with this PR.

Reply:
It just occurred to me, there is a period of time where new clients will be making HEAD requests against relays that may not have the updated to this code which handles them.

What happens in that case?

Reply:
If there is a known error for a missing `HEAD` verb, that could spot the difference between old and new relays. Alternatively, a feature flag, or a 2 phased release is possible. A feature flag is more controversial, I imagine.

Reply:
I've been running catchup locally using my branch (as the client) against betanet which obviously has no peers running the new server code.

What happens is that it just returns 200 along with the entire payload which just gets ignored and then fetched again during the actual catchup. 

So this will cause double the data transfer until the ledgerService is running the new code--which seems ok to me.